packages/zoe/src/contractSupport/percentMath.js

```diff
@@ -0,0 +1,56 @@
+import './types';
+import { assert } from '@agoric/assert';
+import { natSafeMath } from './index';
```

Comment:
should we import it from the file directly?

Reply:
yeah, good idea.